boefjes/boefjes/config.py

```diff
@@ -11,21 +11,21 @@ class RuntimeConfiguration(Enum):
 
 class Settings(BaseSettings):
     base_dir: Path = Path(__file__).parent.resolve()
-    log_cfg: Path = Path(__file__).parent / "logging.json"
+    log_cfg: Path = Path(__file__).parent / "logging.json"  # Follow-up ticket to make logging the same for all modules?
 
     # Worker configuration
-    pool_size: int = 2
-    poll_interval: float = 1.0
+    pool_size: int = 2  # Which value should be the default and why?
+    poll_interval: float = 1.0  # Which value should be the default and why?
```

Comment:
I think pool size of 2 is good for small installations.

I think we should be able to increase the poll interval to 10 without any problems and this lowers the number of requests to the scheduler.